state/rds-postgres.yaml

```diff
@@ -103,6 +103,10 @@ Parameters:
     Type: String
     Default: true
     AllowedValues: [true, false]
+  DBParameterGroupName:
```

Comment:
Please don't forget to add the new parameter to the meat data at the top as well.